app/assets/stylesheets/custom.css.scss

```diff
@@ -152,3 +152,39 @@ p {
     color: #333;
   }
 }
+
+// forms
+
+input,
+textarea,
+select,
+.uneditable-input {
+  @include box-sizing;
+  border: 1px solid #bbb;
+  margin-bottom: 15px;
+  width: 100%;
+}
+
+input {
+  height: auto !important;
+}
+
+label.required:after {
+  color: $purple;
+  content:" *";
+}
+
+#error_explanation {
+  background: rgba(245, 168, 179, 0.54);
+  border: 3px solid rgba(239, 121, 132, 0.9);
+  color: #ff0000;
+  margin-bottom: 2%;
+  .alert.alert-error {
+    margin-bottom: 1%;
+  }
+  ul {
+    color: red;
+    margin: 0 0 10px 0;
```

Comment:
Properties should be ordered color, list-style, margin<br>Shorthand form for property `margin` should be written more concisely as `0 0 10px` instead of `0 0 10px 0`